### DISCUSSION TITLE
[Unstable] Non-crew vitality slider breaks goodsamaritan event

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13735] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

[It's the event where you find unconscious od guy and can treat him with naloxone or leave](https://barotraumagame.com/wiki/Events#Good_Samaritan)
The problem is: you make him go unconscious by giving him fixed amount of affliction, but with vitality mult > 1 he won't go unconscious or die if you leave him

https://github.com/FakeFishGames/Barotrauma/assets/122838333/c27407d4-08cb-4caf-9bba-b15264b9d489

### Reproduction steps

Set non-crew vitality to 2
spam "triggerevent goodsamaritan"

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Vykn** [comment]
Might be worth going through all the events that use \<AfflictionAction\> and checking that they work, rather than playing whack a mole with the issues when players report them

**Comment by Regalis11** [comment]
Thank you for the report! [I created an issue ticket about the bug](https://github.com/FakeFishGames/Barotrauma/issues/13748), we'll look into it!

**Comment by Regalis11** [comment]
Fixed in our private repo now, fix coming up in the next unstable build!
